sale_stock_mto_as_mts_orderpoint/data/stock_data.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <data noupdate="1">
```

Comment:
Just wondering if this file will have any effect with the noupdate flag?

Reply:
hello @grindtildeath , I think the given record will be updated, because Odoo always updates record at the installation step, `noupdate` only effects to the upgrade step